File: backend/src/main/java/com/abel/eventos/infrastructure/adapter/in/web/dto/SesionResponse.java
package com.abel.eventos.infrastructure.adapter.in.web.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class SesionResponse {

    private Long id;
    private Long eventoId;
    private String estado;
    private List<AsientoResponse> asientosSeleccionados;
    private String mensaje;
}

File: mobile/composeApp/src/commonMain/kotlin/org/abel/mobile/data/model/SesionResponse.kt
package org.abel.mobile.data.model

import kotlinx.serialization.Serializable

@Serializable
data class SesionResponse(
    val id: Long? = null,
    val eventoId: Long? = null,
    val estado: String? = null,
    val asientosSeleccionados: List<Asiento>? = null,
    val mensaje: String? = null
)